app/views/groups/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="groups" class="groups">
+  <% if @groups.count.zero? %>
+    <p class="center-message"><em>No groups available for the moment</em></p>
+  <% else %>
+    <% @groups.each do |group| %>
+      <%= render group, group: group %>
+    <% end %>
+  <% end %>
+</div>
+
+<div class="add-category">
+  <%= link_to "Add category", new_group_path, class: "splash-login add-category-button" %>
```

Comment:
- [ ] As pointed out above, please ensure that the `Add category` button sticks to the bottom of the page.

Reply:
@AnselemOdims Also fixed the add category button to be at the bottom. 👍🏼 

<img src="https://user-images.githubusercontent.com/97407840/195899944-d5ccdf1d-8c27-4d16-85d3-107d70de3683.png" width="250">